.gitignore

```diff
@@ -16,4 +16,6 @@ tags.lock
 
 recipe.json
 
-gethdata
\ No newline at end of file
+gethdata
+
+*.patch
```

Comment:
not sure if this will work, do we need `**/upstream.patch`?

Reply:
it will work.